ethportal-api/src/beacon.rs

```diff
@@ -78,16 +78,13 @@ pub trait BeaconNetworkApi {
         -> RpcResult<ContentInfo>;
 
     /// Lookup a target content key in the network
```

Comment:
I think it would be good to clarify that it will first check local storage.
Also in other places.